src/seer/automation/autofix/tools.py

```diff
@@ -44,27 +44,46 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.cleanup()
 
+    def _get_repo_names(self) -> list[str]:
+        if isinstance(self.context, AutofixContext):
+            return [f"{repo.owner}/{repo.name}" for repo in self.context.repos]
```

Comment:
`RepoDefinition` has `.full_name` that gives the full name
```suggestion
            return [repo.full_name for repo in self.context.repos]
```